lib/pharos/up_command.rb

```diff
@@ -79,8 +84,10 @@ def build_config(config_hash)
 
       config = Pharos::Config.new(schema)
 
+      # inject api_endpoint to each host object
+      config.hosts.each { |h| h.api_endpoint = config.api&.endpoint }
```

Comment:
Not really happy with this but we can refactor internals later.

Reply:
Yes, this would be related to #225